test/extensions/filters/common/lua/lua_test.cc

```diff
@@ -19,7 +19,7 @@ namespace Common {
 namespace Lua {
 namespace {
 
-class TestObject : public BaseLuaObject<TestObject> {
+class alignas(32) TestObject : public BaseLuaObject<TestObject> {
```

Comment:
Not sure why this is needed. Can you add a comment?

Reply:
done